Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM gradle:7.4.2-jdk8 AS builder
+WORKDIR /build
+COPY . .
+
+RUN tr -d '\015' <gradlew >gradlew-unix && \
+    chmod +x gradlew-unix && \
+    ./gradlew-unix jar
+
+
+FROM lwieske/java-8:server-jre-8u202-slim
```

Comment:
I think it would be better to use `openjdk:8-jre-slim`, at least it's official release.

I know developers recommend using 8u202 because it's the last free version, and although the official image of openjdk doesn't release a Linux-based docker version of java 8u202, we can use `azul/zulu-openjdk:8u202` or other java distributions.

Reply:
I do heard about openjdk alternative, and java license changes...

I just followed README instruction, since I'm not really familiar with Java.
and really don't know why we can't just use newest OpenJDK :P

Anyway, I'll do as you say :D